tests/testStreamOpen.cpp

```diff
@@ -389,6 +391,27 @@ TEST_F(StreamOpenOutput, LowLatencyStreamHasSmallBufferSize){
     }
 }
 
+// Make sure the parameters get copied from the child stream.
+TEST_F(StreamOpenOutput, AAudioOutputSampleRate44100FilterConfiguration) {
+    if (mBuilder.isAAudioRecommended()) {
+        mBuilder.setDirection(Direction::Output);
+        mBuilder.setPerformanceMode(PerformanceMode::LowLatency);
+        mBuilder.setSharingMode(SharingMode::Exclusive);
+        // Try to force the use of a FilterAudioStream by requesting conversion.
+        mBuilder.setSampleRate(44100);
+        mBuilder.setSampleRateConversionQuality(SampleRateConversionQuality::Medium);
+        ASSERT_TRUE(openStream());
+        ASSERT_LT(0, mStream->getHardwareSampleRate());
```

Comment:
getHardware* functions are only supported in Android API 34+. Please wrap this test around an SDK level check.